crates/cansat-stm32f4/src/startup.rs

```diff
@@ -149,11 +146,31 @@ fn init_bme280(i2c: I2c1Proxy, delay: &mut Delay) -> Result<Bme280, Bme280Error>
     Ok(bme280)
 }
 
-fn init_gps(mut serial: Serial1) -> Gps {
+fn init_gps(mut serial: Serial1) -> Result<Gps, GpsError> {
     defmt::info!("Initializing GPS");
 
     serial.listen(serial::Event::Rxne);
-    Gps::new(serial)
+    let mut gps = Gps::new(serial);
+
+    let set_gll_output_rate = b"$PUBX,40,GLL,0,0,0,0,0,0*5C\r\n";
+    gps.send(set_gll_output_rate)?;
+
+    let set_gsa_output_rate = b"$PUBX,40,GSA,0,0,0,0,0,0*4E\r\n";
+    gps.send(set_gsa_output_rate)?;
+
+    let set_gsv_output_rate = b"$PUBX,40,GSV,0,0,0,0,0,0*59\r\n";
+    gps.send(set_gsv_output_rate)?;
+
+    let set_gga_output_rate = b"$PUBX,40,GGA,0,1,0,0,0,0*5B\r\n";
+    gps.send(set_gga_output_rate)?;
+
+    let set_vtg_output_rate = b"$PUBX,40,VTG,0,0,0,0,0,0*5E\r\n";
+    gps.send(set_vtg_output_rate)?;
+
+    let set_rmc_output_rate = b"$PUBX,40,RMC,0,0,0,0,0,0*47\r\n";
+    gps.send(set_rmc_output_rate)?;
+
+    Ok(gps)
```

Comment:
I haven't find any docs for this, only one [arduino lib](https://github.com/Velleman/VMA430_GPS_Module/tree/master) that uses different values to initialize module. Can you provide link to docs or explain those values? You already tested this, so it probably works, but im curious :P

Reply:
[25.3](https://content.u-blox.com/sites/default/files/products/documents/u-blox7-V14_ReceiverDescriptionProtocolSpec_%28GPS.G7-SW-12001%29_Public.pdf)